pkg/operator/configobserver/config_observer_controller.go

```diff
@@ -87,7 +91,9 @@ func (c ConfigObserver) sync() error {
 	spec := originalSpec.DeepCopy()
 	// don't worry about errors.  If we can't decode, we'll simply stomp over the field.
 	existingConfig := map[string]interface{}{}
-	json.NewDecoder(bytes.NewBuffer(spec.ObservedConfig.Raw)).Decode(&existingConfig)
+	if err := json.NewDecoder(bytes.NewBuffer(spec.ObservedConfig.Raw)).Decode(&existingConfig); err != nil {
+		glog.V(4).Infof("decode of existing config failed with error: %v", err)
```

Comment:
We can make this noiser if you like.  Just don't fail on it.

Reply:
@deads2k i don't think we need, new goland version highlight unhandled errors and it triggered my OCD ;-)